lib/src/test/java/tech/relaycorp/relaydroid/test/ParcelDeliveryCertificates.kt

```diff
@@ -0,0 +1,29 @@
+package tech.relaycorp.relaydroid.test
+
+import tech.relaycorp.relaynet.issueEndpointCertificate
+import tech.relaycorp.relaynet.issueGatewayCertificate
+import tech.relaycorp.relaynet.testing.pki.KeyPairSet
+import java.time.ZonedDateTime
+
+object ParcelDeliveryCertificates {
```

Comment:
Please use [`PDACertPath`](https://github.com/relaycorp/relaynet-jvm-testing#mock-certification-paths) instead of this object.